custom/conf/app.example.ini

```diff
@@ -1221,6 +1221,9 @@ LEVEL = Info
 ;; For custom reactions, add a tightly cropped square image to public/assets/img/emoji/reaction_name.png
 ;REACTIONS = +1, -1, laugh, hooray, confused, heart, rocket, eyes
 ;;
+;; Change the users witch are displayed on mouse hover over (via tooltip) on reactions.
```

Comment:
I think the information that this is the number of users is missing. What about:

Change the number of users that are displayed in reactions tooltip (triggered by mouse hover)

Reply:
I'm all  for better wording suggestions